packages/wdio-allure-reporter/tests/suite.test.ts

```diff
@@ -157,15 +200,14 @@ describe('Failed tests', () => {
         reporter.onSuiteEnd(suiteEnd())
         reporter.onRunnerEnd(runnerEnd())
 
-        const results = getResults(outputDir)
-        expect(results).toHaveLength(1)
-        allureXml = results[0]
-
-        expect(allureXml('test-case > name').text()).toEqual('should can do something')
-        expect(allureXml('test-case').attr('status')).toEqual('failed')
+        const { results } = getResults(outputDir)
+        const browserParameter = results[0].parameters.find((param: Parameter) => param.name === 'browser')
 
-        expect(allureXml('test-case parameter[kind="argument"]')).toHaveLength(1)
-        expect(allureXml('test-case parameter[name="browser"]').eq(0).attr('value')).toEqual(testStart().cid)
+        expect(results).toHaveLength(1)
```

Comment:
please assert length before you get value in line 204

Reply:
:+1: